app/(components)/TicketCard.jsx

```diff
@@ -11,10 +11,11 @@ const TicketCard = ({ ticket }) => {
       hour: "2-digit",
       minute: "2-digit",
       hour12: true,
+      timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone
     };
 
     const date = new Date(timestamp);
-    const formattedDate = date.toLocaleString("en-US", options);
+    const formattedDate = date.toLocaleString("en-IN", options);
```

Comment:
why not fetch the locale string "en-IN" also form Intl

Reply:
Can be done, in issue it was specifically mentioned en-IN, hence i specifically made it for en-IN.
Shoul i change it now??

Reply:
yes please do